GccBoardProject1/src/numpyInC.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * numpyInC.h
+ *
+ * Created: 3/24/2017 8:59 PM
+ * Author: Udaya Malik
+ */ 
+
+#ifndef NUMPYINC_H_
+#define NUMPYINC_H_
+
+#include <asf.h>
+#include <string.h>
+
+void init_uart(void);
+void uart_out(const uint8_t *string, uint16_t length);
+static uint16_t frame_median[NUM_PIXELS];
+static uint16_t array[NUM_RAW_FRAMES];
```

Comment:
You may want to reconsider if these really should be in the header file, because it makes them effectively global. Consider:
 - Does anything else outside of this file use this static variable?
 - Does this state necessarily need to be preserved outside of a function call. For example, can you make it a local variable in a calling function, which you pass as a pointer to the child function?